<?xml version="1.0"?>
<launch>
    <arg name="cam_1" default="on"/>
    <arg name="cam_2" default="on"/>
    <arg name="cam_3" default="on"/>
    <arg name="cam_4" default="on"/>
    <arg name="cam_5" default="on"/>
    <arg name="calib_mode" default="false"/>
    <include file="$(find kuka_cam)/launch/individual_cam.launch" if="$(eval cam_1 == 'on')">
        <arg name="cam_num" value="1"/>
        <arg name="calib_mode" value="$(arg calib_mode)"/>
    </include>
    <include file="$(find kuka_cam)/launch/individual_cam.launch" if="$(eval cam_2 == 'on')">
        <arg name="cam_num" value="2"/>
        <arg name="calib_mode" value="$(arg calib_mode)"/>
    </include>
    <include file="$(find kuka_cam)/launch/individual_cam.launch" if="$(eval cam_3 == 'on')">
        <arg name="cam_num" value="3"/>
        <arg name="calib_mode" value="$(arg calib_mode)"/>
    </include>
    <include file="$(find kuka_cam)/launch/individual_cam.launch" if="$(eval cam_4 == 'on')">
        <arg name="cam_num" value="4"/>
        <arg name="calib_mode" value="$(arg calib_mode)"/>
    </include>
    <include file="$(find kuka_cam)/launch/individual_cam.launch" if="$(eval cam_5 == 'on')">
        <arg name="cam_num" value="5"/>
        <arg name="calib_mode" value="$(arg calib_mode)"/>
    </include>

    <param name="cam_1_on" type="bool" value="$(eval cam_1 == 'on')" />
    <param name="cam_2_on" type="bool" value="$(eval cam_2 == 'on')" />
    <param name="cam_3_on" type="bool" value="$(eval cam_3 == 'on')" />
    <param name="cam_4_on" type="bool" value="$(eval cam_4 == 'on')" />
    <param name="cam_5_on" type="bool" value="$(eval cam_5 == 'on')" />
</launch>
